cub/cub/device/dispatch/tuning/tuning_transform.cuh

```diff
@@ -264,39 +267,58 @@ struct policy_hub<RequiresStableAddress, ::cuda::std::tuple<RandomAccessIterator
     using algo_policy = ::cuda::std::_If<use_fallback, prefetch_policy_t<256>, default_vectorized_policy_t>;
   };
 
-  // TODO(bgruber): should we add a tuning for 750? They should have items_per_thread_from_occupancy(256, 4, ...)
+  struct policy800 : ChainedPolicy<800, policy800, policy300>
+  {
+  private:
+    static constexpr int block_threads = 256;
+    using async_policy                 = async_copy_policy_t<block_threads, ldgsts_size_and_align>;
+    // We cannot use the architecture-specific amount of SMEM here instead of max_smem_per_block, because this is not
+    // forward compatible. If a user compiled for sm_xxx and we assume the available SMEM for that architecture, but
+    // then runs on the next architecture after that, which may have a smaller available SMEM, we get a crash.
+    static constexpr bool exhaust_smem =
+      memcpy_async_smem_for_tile_size<RandomAccessIteratorsIn...>(
+        block_threads * async_policy::min_items_per_thread, ldgsts_size_and_align)
+      > int{max_smem_per_block};
+    static constexpr bool use_fallback =
+      RequiresStableAddress || !can_memcpy_inputs || no_input_streams || exhaust_smem;
```

Comment:
Removed the check for overaligned types for sm_80, since it was already supported.